spikeinterface/comparison/multisortingcomparison.py

```diff
@@ -335,7 +207,7 @@ def __init__(self, sampling_frequency, multisortingcomparison,
 
         BaseSorting.__init__(self, sampling_frequency=sampling_frequency, unit_ids=unit_ids)
         if len(unit_ids) > 0:
-            for k in ('agreement_number', 'avg_agreement', 'sorter_unit_ids'):
+            for k in ('agreement_number', 'avg_agreement', 'unit_ids'):
```

Comment:
No! unit_ids cannot be a property for a sorting. sorter_unit_ids was a good choice because it was referring to old unit ids.

Reply:
Not really because for TemplateComparison units don't belong to a sorter but rather to a session